try to add some info to the organizer guide (#9)
jhudsl/AnVIL_CoFests_2024: organizer-guide.Rmd

## CoFests! Goals {- .unlisted}

- Build an inclusive community that is supportive of newcomers
- Contribute to [open science](https://en.wikipedia.org/wiki/Open_science)
- Produce a final product that accomplishes one of the following for a topic
  - Makes useful forward progress
  - Identifies opportunities and boundaries or limitations

> The final product of CoFests doesn't need to be code! These Collaboration Fests can support story-boarding future work or development of documentation or training materials. If it's something that will help the community and make progress on an AnVIL associated topic and there's sufficient interest, the idea is game! [As Galaxy phrased it for their CoFests:](https://galaxyproject.org/events/gcc2023/cofest/#goals)

Reviewer: ```suggestion
## Final Product {-}

The final product of CoFests doesn't need to be code! These Collaboration Fests can support story-boarding future work or development of documentation or training materials. If it's something that will help the community and make progress on an AnVIL associated topic and there's sufficient interest, the idea is game! [As Galaxy phrased it for their CoFests:](https://galaxyproject.org/events/gcc2023/cofest/#goals)
```

> [The goal of collaboration fests is to expand the ecosystem] Not just the code, but the whole ecosystem. That includes training, tools, best practice workflows, documentation, test cases, translations, infrastructure, and yes, even code.  

Prior to the CoFests, the scientific and AnVIL co-leads will work together to enumerate some ideas for final products and post them publicly in this book.

## Logistics {- .unlisted}

- 3 Parallel tracks, 5+ hours across Nov 12/13
  - Scalable Machine Learning and Data Science
  - Polygenic Risk Score Analysis
  - Wrap a Tool (+ Intro AnVIL/WDL)
- Scientific Lead and AnVIL Outreach Co-Lead for support
- We expect 4-10 participants per track

## Example Schedule for a Track {- .unlisted}

> Day 1 (2.5 hrs scheduled)

Reviewer: ```suggestion
### Day 1 (2.5 hrs scheduled)
```

| Task                      | Suggested time | Lead                  | Description |
|:-------------------------:|:---------------|:----------------------|:------------|
| Problem statement         | 20-30 minutes  | Scientific Lead       | Provide background info on the track's overall topic and may include a brief summary of the pre-worked collaboration ideas.|
| Brainstorming             | 15-30 minutes  | Both                  | What project will the track tackle? Focus on understanding the participant backgrounds and skills and matching those with track ideas and actionable tasks to accomplish the goal. |
| Onboarding                | 30-60 minutes  | AnVIL Outreach Co-lead| Handle billing accounts and other logistics necessary to accomplish the work |
| Work-regroup, work-regroup| 30-60 minutes  | --                    | Formal collaboration work time left before dinner. (Groups may choose to work together informally after dinner too) |

> Day 2 (3 hrs scheduled)

Reviewer: ```suggestion
### Day 2 (3 hrs scheduled)
```

| Task                      | Suggested time | Lead                  | Description |
|:-------------------------:|:---------------|:----------------------|:------------|

> | Work-regroup, work-regroup| 2 - 2.5 hrs    | --                    | Formal collaboration work time (with a coffee break) |

Reviewer 2: ```suggestion
| Work-regroup | 2 - 2.5 hrs    | --                    | Formal collaboration work time (with a coffee break) |
```

Or is "Work-regroup, work-regroup" actually meant to be?

Author: It's a phrase that @cutsort used in our meeting and the original draft and I kept it

## CoFests! Goals {- .unlisted}

- Build an inclusive community that is supportive of newcomers

> - Contribute to [open science](https://en.wikipedia.org/wiki/Open_science)

Reviewer: ```suggestion
- Contribute to [open science](https://en.wikipedia.org/wiki/Open_science) and collaboration
```

> - Contribute to [open science](https://en.wikipedia.org/wiki/Open_science)

Author: ```suggestion
- Collaborate and contribute to [open science](https://en.wikipedia.org/wiki/Open_science)
```

## CoFests! Goals {- .unlisted}

- Build an inclusive community that is supportive of newcomers

> - Contribute to [open science](https://en.wikipedia.org/wiki/Open_science)

Reviewer 2: ```suggestion
- Contribute to [open science](https://en.wikipedia.org/wiki/Open_science) for open collaboration. 
```
Could a main goal of Cofests be the collaborative experience? So maybe specifying one of the key aspects of open science could be necessary.

Author: Ah, I really like the of pointing out collaboration is a key aspect of open science! I'll slightly rephrase this but very grateful for the suggestion and stressing the importance of collaboration. Thanks!

- Report out a final product that accomplishes one of the following for a topic
  - Makes useful forward progress
  - Identifies opportunities and boundaries or limitations

The final product of CoFests doesn't need to be code! These Collaboration Fests can support story-boarding future work or development of documentation or training materials. If it's something that will help the community and make progress on an AnVIL associated topic and there's sufficient interest, the idea is game! [As Galaxy phrased it for their CoFests:](https://galaxyproject.org/events/gcc2023/cofest/#goals)

> [The goal of collaboration fests is to expand the ecosystem] Not just the code, but the whole ecosystem. That includes training, tools, best practice workflows, documentation, test cases, translations, infrastructure, and yes, even code.  

Prior to the CoFests, the scientific and AnVIL co-leads will work together to enumerate some ideas for final products and post them publicly in this book.

## Logistics {- .unlisted}

- 3 Parallel tracks, 5+ hours across Nov 12/13
  - Scalable Machine Learning and Data Science
  - Polygenic Risk Score Analysis
  - Wrap a Tool (+ Intro AnVIL/WDL)

> - Scientific Lead and AnVIL Co-Lead for support

Reviewer: ```suggestion
- Scientific Lead and AnVIL Outreach Co-Lead for support
```

> - We expect perhaps 4-10 participants per track

Reviewer: ```suggestion
- We expect 4-10 participants per track
```

## Example Schedule for a Track {- .unlisted}

- First Day (2.5 hrs scheduled):

>   - Problem statement (perhaps with a short demo) by scientific lead

Reviewer: ```suggestion
  - Problem statement by scientific lead
```

>     This part of the session will provide background info on the track's overall topic and may include a brief summary of the pre-worked ideas. Perhaps 20-30 minutes?

Reviewer: ```suggestion
    This part of the session will provide background info on the track's overall topic and may include a brief summary of the pre-worked ideas. [Suggested time: 20-30 minutes]
```

  - Brainstorming

    What project will the track tackle? This should be quicker then the problem statement, focusing on understanding the participant backgrounds and skills and matching those with track ideas and actionable tasks to accomplish the goal.

  - Onboarding by AnVIL co-lead

    Handle billing accounts and other logistics necessary to accomplish the work.

  - Work-regroup, work-regroup

>     Probably half an hour to an hour of formal work time left on this first day before dinner. Groups may choose to work together, informally after dinner.

Reviewer: Would love for this to be in table format!

Reviewer 3: I like the table idea

Author: Sounds good -- will try to mock one up

Reviewer 2: I also like the table idea

- Second Day (3 hrs scheduled):

>   - Work-regroup, work-regroup

Reviewer 2: ```suggestion
  - Work-regroup
```

  - Construct a "report out"

    The report out may be a Google doc, Google slides, a GitHub repo, [a website](https://github.com/jhudsl/OTTR_Template_Website), an [OTTR book](https://github.com/jhudsl/OTTR_Template), etc.

> - Share report out during the CoFest Recap Session!  

Author: ```suggestion
- Share final report during the CoFest Recap Session!  
```

> - Share report out during the CoFest Recap Session!  

Reviewer 3: Maybe just "share final report during the CoFest Recap Session!"?
